apps/common/test/lexical/ast/env_test.exs

```diff
@@ -293,17 +293,17 @@ defmodule Lexical.Ast.EnvTest do
         }
       ]
       env = new_env(source)
-      assert in_context?(env, :struct_arguments)
+      assert in_context?(env, :struct_fields)
     end
 
     test "is true even if the value of a struct key is a tuple" do
       env = new_env("%User{favorite_numbers: {3}|")
-      assert in_context?(env, :struct_arguments)
+      assert in_context?(env, :struct_fields)
     end
 
     test "is true even if the cursor is at a nested struct" do
       env = new_env("%User{address: %Address{}|")
-      assert in_context?(env, :struct_arguments)
+      assert in_context?(env, :struct_fields)
```

Comment:
While I know that the ancestor of map will never have {%, _, _}, I thought it might be better to add a test here:

```elixir
    test "is false if the cursor is in a map" do
      env = "%{|field: value}" |> wrap_with_module() |> new_env()
      refute in_context?(env, :struct_fields)
    end
```